anchore_engine/analyzers/syft.py

```diff
@@ -48,6 +119,17 @@ def handle_python(findings, artifact):
 
 
 def handle_gem(findings, artifact):
+    """Gem results handler for syft output.
+
+    Args:
```

Comment:
It is great to have more thorough docstrings, but there is no convention in Anchore Engine for them and this is adding another variation of the existing ones. 

Since there is no convention yet, can this PR avoid making it specific?

Reply:
I'll remove them for now.